warehouse/query-core/src/main/java/datawave/query/index/lookup/ConcurrentScannerInitializer.java

```diff
@@ -35,6 +35,12 @@ public ConcurrentScannerInitializer(BaseIndexStream stream) {
     public BaseIndexStream call() throws Exception {
         if (stream.context() == StreamContext.INITIALIZED) {
             if (stream.hasNext()) {
+                if (stream instanceof ScannerStream) {
+                    // the Intersection and Union classes are predicated on the accuracy of the StreamContext,
+                    // so update this stream's context from INITIALIZED to PRESENT
+                    stream.context = StreamContext.PRESENT;
+                    return stream;
+                }
                 return ScannerStream.withData(stream, stream.currentNode());
```

Comment:
Does not return ScannerStream.withData(stream, stream.currentNode()) already return a stream with a context of PRESENT?

Reply:
The case addressed here is
`
RangeStream line 503 -> visit(EQ) -> return ScannerStream.initialized(scannerSession, entryParser, node);
ScannerStream line 109 -> initial context is INITIALIZED
`